src/module.c

```diff
@@ -13532,6 +13535,111 @@ int RM_GetDbIdFromDefragCtx(RedisModuleDefragCtx *ctx) {
     return ctx->dbid;
 }
 
+/* --------------------------------------------------------------------------
+ * ## Key API for Set type
+ *
+ * See also RM_ValueLength(), which returns the cardinality of a set.
+ * -------------------------------------------------------------------------- */
+
+/* Add new elements into a set.
+ *
+ * Returns REDISMODULE_OK if an element has been added. On failure,
```

Comment:
This comment isn't correct, if all the elements already exist in the set, `numadded` will be 0.

Reply:
A bit difficult to describe it more accurate. So changed it to "_Returns REDISMODULE_OK if elements have been added successfully._". Mean no failure happened though zero element to be added.